platform/podjs/src/browserPod.ts

```diff
@@ -358,32 +355,83 @@ class IDBPolyOut implements PolyOut {
         throw "Not implemented: writeFile";
     }
 
-    async importArchive(url: string): Promise<string> {
-        const { data: dataUrl, fileName } = FileUrl.fromUrl(url);
-        const blob = await (await fetch(dataUrl)).blob();
+    /// destUrl should be the same as zipId
+    async importArchive(url: string, destUrl?: string): Promise<string> {
+        const { fileName } = FileUrl.fromUrl(url);
+        const buffer = await (await fetch(url)).arrayBuffer();
```

Comment:
This is a regression. Since this change got merged importing an archive in *podjs* fails with follwing error being logged in the console:

    pod.js:9355 GET data:application/zip;base64,UEsDBBQAAAAAAAD43p0AAAAAAAAAAAAAAAAaAAAAZ29vZ2x…1sUEsFBgAAAACaAJoA9UoAAAE5cAAAAA==/google-fake-malcomgerlach-2022-3-22.zip net::ERR_INVALID_URL

Note that previously we called `fetch(FileUrl.fromUrl(url).data)` but now you are passing the `url` unchanged to `fetch()`.